src/schema.js

```diff
@@ -560,6 +638,7 @@ const variables = {
     },
     backend_requests: {
         child: 'default',
+        description: "An object with all backend requests and their attributes. To access a specific request use the related label.\n\n`request` and `proxy` blocks without a label will be available as `default`.\n\n**Example:** Use `backend_requests.default.method` to access the HTTP method of the default  request.",
```

Comment:
I don't see a font change for code-marked phrases. Do you?

Reply:
I do. There's a black background. But the font stay proportional.

![tooltip](https://user-images.githubusercontent.com/1311513/161243556-e88a1cec-8894-437a-900d-706c5f15f836.png)
.